docs/api_requests/block-vhost-user.md

```diff
@@ -1,5 +1,8 @@
 # Vhost-user block device
 
+Support is currently in **developer preview**. See
```

Comment:
We can use some GitHub extended markup to highlight it:

```
> [!WARNING]
> Support is currently in **developer preview**. See ...
```

> [!WARNING]
> Support is currently in **developer preview**. See ...

https://docs.github.com/en/get-started/writing-on-github/getting-started-with-writing-and-formatting-on-github/basic-writing-and-formatting-syntax#alerts

Reply:
thanks, added